client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -21,118 +20,139 @@ class PostRelativeTime extends React.PureComponent {
-				</time>
+				{ time && (
+					<>
+						<Gridicon icon="time" size={ this.props.gridiconSize || 18 } />
+						<time className="post-relative-time-status__time-text" dateTime={ time }>
+							{ this.getDisplayedTimeForLabel() }
+						</time>
+					</>
+				) }
 			</span>
 		);
 	}
 
-	getStatusText() {
+	/**
+	 * Get status label
+	 *
+	 * @param {boolean} onlySticky sends back the "sticky" label. Special case as is using the same template but for is unrelated to the status
+	 *
+	 */
+	getStatusText( onlySticky = false ) {
 		const status = this.props.post.status;
 		let statusClassName = 'post-relative-time-status__status';
 		let statusIcon = 'aside';
 		let statusText;
 
-		if ( this.props.post.sticky ) {
+		if ( onlySticky ) {
 			statusText = this.props.translate( 'sticky' );
 			statusClassName += ' is-sticky';
 			statusIcon = 'bookmark-outline';
 		} else if ( status === 'pending' ) {
 			statusText = this.props.translate( 'pending review' );
 			statusClassName += ' is-pending';
-		} else if ( status === 'future' ) {
-			const moment = this.props.moment;
-			const now = moment();
-			const scheduledDate = moment( this.props.post.date );
-			// If the content is scheduled to be release within a year, do not display the year at the end
-			const scheduledTime = scheduledDate.calendar( null, {
-				sameElse: this.props.translate( 'll [at] LT', {
-					comment:
-						'll refers to date (eg. 21 Apr) & LT refers to time (eg. 18:00) - "at" is translated',
-				} ),
-			} );
-
-			const displayScheduleTime =
-				scheduledDate.diff( now, 'years' ) > 0
-					? scheduledTime
-					: scheduledTime.replace( scheduledDate.format( 'Y' ), '' );
-
-			statusText = this.props.translate( 'scheduled for %(displayScheduleTime)s', {
-				comment: '%(displayScheduleTime)s is when a scheduled post is set to be published',
+		} else if ( status === 'trash' ) {
+			statusClassName += ' is-trash';
+			statusIcon = 'trash';
+			const displayScheduleTime = this.getDisplayedTimeForLabel();
+			statusText = this.props.translate( 'trashed %(displayScheduleTime)s', {
+				comment: '%(displayScheduleTime)s is when a post or page was trashed',
 				args: {
 					displayScheduleTime,
 				},
 			} );
+		} else if ( status === 'future' ) {
 			statusClassName += ' is-scheduled';
 			statusIcon = 'calendar';
-		} else if ( status === 'trash' ) {
-			statusText = this.props.translate( 'trashed' );
-			statusClassName += ' is-trash';
-			statusIcon = 'trash';
-		} else if ( this.props.includeBasicStatus ) {
-			if ( status === 'draft' ) {
-				statusText = this.props.translate( 'draft' );
-			} else if ( status === 'publish' ) {
-				statusText = this.props.translate( 'published' );
-			} else if ( status === 'new' ) {
-				statusText = this.props.translate( 'Publish immediately' );
-			}
+			const displayScheduleTime = this.getDisplayedTimeForLabel();
+			statusText = this.props.translate( 'scheduled %(displayScheduleTime)s', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'scheduled for %(displayScheduleTime)s' )` *ES Score: 8*